ui/app/AppLayouts/Chat/popups/community/CreateChannelPopup.qml

```diff
@@ -75,8 +75,8 @@ StatusDialog {
     }
 
     function isFormValid() {
-        return (nameInput.valid &&
-                descriptionTextArea.valid) &&
+        return (nameInput.valid && nameInput.dirty &&
+                descriptionTextArea.valid && descriptionTextArea.dirty) &&
```

Comment:
Hmm, I'd like to hear the others' opinion but I don't like this behavior change; when something is `valid` I shouldn't need to check for `dirty` as well

Reply:
So situation is tricky
If we use ValidationMode.Always and only valid check - we instantly get errors shown for empty names and Next button blocked
If we use ValidationMode.OnlyWhenDirty and valid check - we dont have errors but Next is enabled with empty names
So the only way is to use OnleWhenDirty check and valid + dirty
Or set valid by default to false

Reply:
And we need to check what default valid = false will lead to

Reply:
Looks like default valid = false is ok 
I even can see that  to overcome default true some places are using construction like this  
`!!displayNameInput.text && displayNameInput.valid` to not allow empty values which are valid :D